CONTRIBUTING.md

```diff
@@ -138,7 +138,7 @@ How add a new feature:
 12. Push your first change branch to know you start to work on.
 
     ```bash
-    $ git push -u your-name 15.0-your_new_feature_theme-your-name
+    $ git push -u your-name 18.0-your_new_feature_theme-your-name
```

Comment:
My bad, I've added these changes to the initialization commit, as well as some missing changes in the repo's README. Please rebase.